src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -253,14 +262,20 @@ details.sd-dropdown {
     // so that it is defined in both places
     --pst-sd-dropdown-color: var(--pst-gray-500);
     --pst-sd-dropdown-bg-color: var(--pst-color-surface);
+
     & + div.sd-summary-content {
+      border: 0;
+
       --pst-sd-dropdown-color: var(--sd-color-card-border);
     }
+
     @each $name in $sd-semantic-color-names {
       &.sd-bg-#{$name} {
         --pst-sd-dropdown-color: var(--sd-color-#{$name});
         --pst-sd-dropdown-bg-color: var(--sd-color-#{$name}-bg);
+
         // Otherwise it won't be defined in the sibling element
+        // stylelint-disable-next-line no-duplicate-selectors
```

Comment:
Did not want to risk moving this around

Reply:
is it possible that stylelint is wrong here?  it's inside an `@each` so I think not collapsible with the similar selector on line 266 or 298

Reply:
Good catch. This was my mistake. The disable-next-line was only needed further down, not here.